src/main/java/me/dinowernli/grpc/prometheus/MonitoringServerCall.java

```diff
@@ -61,11 +62,12 @@ private void reportStartMetrics() {
   }
 
   private void reportEndMetrics(Status status) {
-    serverMetrics.recordServerHandled(status.getCode(), requestMetadata);
+    final Status.Code code = status.getCode();
```

Comment:
nit: we've been somewhat consistently avoiding the clutter of "final" for local variables